kubernetes/config/incluster_config.c

```diff
@@ -24,15 +26,15 @@ static int checkServiceAccountFile(const char *fileName)
     struct stat info;
     if (-1 == stat(fileName, &info)) {
         switch (errno) {
-        case ENOENT:
-            fprintf(stderr, "%s: The file %s does not exist.[%s]\n", fname, fileName, strerror(errno));
-            return -1;
-        case EACCES:
-            fprintf(stderr, "%s: No permission to read the file %s.[%s]\n", fname, fileName, strerror(errno));
-            return -1;
-        default:
-            fprintf(stderr, "%s: Cannot retrieve the information of file %s.[%s]\n", fname, fileName, strerror(errno));
-            return -1;
+            case ENOENT:
+                fprintf(stderr, "%s: The file %s does not exist.[%s]\n", fname, fileName, strerror(errno));
```

Comment:
Do you have a tool to format the souce code ?

Reply:
Yes. I use CLion.

Reply:
We use a code style format tool https://github.com/kubernetes-client/c/blob/master/code-check/code-style-check.sh in this repo to format all *.c files except generated files. Let's keep consistent as before.

You could check it by 
```shell
sh ./code-check/code-style-check.sh kubernetes/config/incluster_config.c
```

Reply:
Done. Btw, this script doesn't work with CRLF endings.